languages/en-US/moderation/setwarns.json

```
@@ -2,11 +2,11 @@
     "DESCRIPTION": "Define the sanctions that members will get after a certain number of warns!",
     "USAGE": "{{prefix}}setwarns [kick/ban] [number/reset]",
     "EXAMPLES": "{{prefix}}setwarns kick 10\n{{prefix}}setwarns ban 10\n{{prefix}}setwarns ban reset",
-    "MISSING_TYPE": "Please specify a valid sanction type! (`kick`, `ban`)",
-    "SUCCESS_KICK": "Configuration saved! When a member will reach **{{count}}** warns, they will be kicked. Type `{{prefix}}configuration` to see your new configuration!",
-    "SUCCESS_BAN": "Configuration saved! When a member will reach **{{count}}** warns, he will be banned. Type `{{prefix}}configuration` to see your new configuration!",
-    "SUCCESS_KICK_RESET": "Configuration saved! The kick sanction is no longer automatic! Type `{{prefix}}configuration` to see your new configuration!",
-    "SUCCESS_BAN_RESET": "Configuration saved! The ban sanction is no longer automatic! Type `{{prefix}}configuration` to see your new configuration!",
-    "AUTO_BAN": "**{{username}}** was automatically banned because they reach more than **{{count}}** warns!",
-    "AUTO_KICK": "**{{username}}** was automatically kicked because they reach more than **{{count}}** warns!"
+    "MISSING_TYPE": "Please specify sanction between `kick` and `ban`",
+    "SUCCESS_KICK": "Configuration saved! When a member will reach **{{count}}** warns, he will be kicked.\nType `{{prefix}}configuration` to see your new configuration!",
+    "SUCCESS_BAN": "Configuration saved! When a member will reach **{{count}}** warns, he will be banned.\nType `{{prefix}}configuration` to see your new configuration!",
+    "SUCCESS_KICK_RESET": "Configuration saved! The `kick` sanction is no longer automatic!\nType `{{prefix}}configuration` to see your new configuration!",
+    "SUCCESS_BAN_RESET": "Configuration saved! The `ban` sanction is no longer automatic!\nType `{{prefix}}configuration` to see your new configuration!",
```

Même suggestion que plus bas, peut-être que `send` est plus précis ici.
```suggestion
    "SUCCESS_KICK": "Configuration saved! When a member will reach **{{count}}** warns, he will be kicked.\nSend `{{prefix}}configuration` to see your new configuration!",
    "SUCCESS_BAN": "Configuration saved! When a member will reach **{{count}}** warns, he will be banned.\nSend `{{prefix}}configuration` to see your new configuration!",
    "SUCCESS_KICK_RESET": "Configuration saved! The `kick` sanction is no longer automatic!\nSend `{{prefix}}configuration` to see your new configuration!",
    "SUCCESS_BAN_RESET": "Configuration saved! The `ban` sanction is no longer automatic!\nSend `{{prefix}}configuration` to see your new configuration!",
```

---

En fait ici il faudrait mettre une formulation similaire à celle pour les commandes à plusieurs étapes (FORM_1: https://github.com/Androz2091/AtlantaBot/blob/00642c5ab0f427d4de8e691380674f33603a5959/languages/en-US/administration/welcome.json).
Le seul problème c'est que cette formulation que je trouve vraiment pas mal, j'en ai eu l'idée pendant ce chantier. Ce qui veut dire qu'il y en a sûrement d'autre à changer.

---

ça donnerait "See your new configuration by sending `{{prefix}}configuration`!" ? Si oui, ça peut mais la question c'était plus sur le verbe à utiliser. Il y a `send`, `type` et `enter`... Et là j'ai pas un niveau d'expertise suffisant pour choisir quel est le mieux. Pour moi ce serait `send` mais bon...

---

5dd458f0fe34163afc1280264204bc0635584210

---

3440edbb6fd8b07193e37a6a96109025b03be7b9